cmd/json-schema/main.go

```diff
@@ -0,0 +1,61 @@
+// dagger-json-schema is a tool to generate json schema from Dagger module config
+// struct.
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"slices"
+
+	"github.com/dagger/dagger/core/modules"
+	"github.com/invopop/jsonschema"
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:  "json-schema",
+	RunE: generateSchema,
+	Args: cobra.ExactArgs(1),
+}
+
+func generateSchema(cmd *cobra.Command, args []string) error {
+	for _, target := range targets {
+		if !slices.Contains(args, target.id) {
+			continue
+		}
+
+		r := new(jsonschema.Reflector)
+		if err := r.AddGoComments("github.com/dagger/dagger", "./core/modules"); err != nil {
+			return err
+		}
+
+		s := r.Reflect(&modules.ModuleConfig{})
```

Comment:
It needs to be:

```suggestion
		if err := r.AddGoComments("github.com/dagger/dagger", target.path); err != nil {
			return err
		}

		s := r.Reflect(target.value)
```

??

Reply:
Bleh, had these changes locally, but somehow missed committing them :sweat_smile: